README.md

```diff
@@ -60,6 +60,21 @@ baldinof_road_runner:
   default_integrations: false
 ```
 
+## Doctrine connection handler
+Due to the fact that roadrunner assumes that the process works in demonized mode, there may be problems with disconnecting the database, this problem is handled in this bundle if a doctrine is used to connect to the database.
+
+By default, in the event of a connection failure, the process will be stopped and the request will end with a 500 http code, to prevent this and to correctly process failed connections, without erroneous answers - you need to add [symfony/proxy-manager-bridge](https://github.com/symfony/proxy-manager-bridge) to your project:
+```
+composer require symfony/proxy-manager-bridge
+```
+
+## Sessions in database
+In accordance with the problem described above, to store sessions in the database, you should use the doctrine connection, for example, you can use the [shapecode/doctrine-session-handler-bundle](https://github.com/shapecode/doctrine-session-handler-bundle) bundle
```

Comment:
Maybe mention [this](https://symfony.com/doc/current/session/database.html#store-sessions-in-a-relational-database-mysql-postgresql)?

I tried it and it works, but the PDO instance is recreated on each request.

Also it could be a good addition in the future to have a copy of [PdoSessionHandler](https://github.com/symfony/symfony/blob/5.1/src/Symfony/Component/HttpFoundation/Session/Storage/Handler/PdoSessionHandler.php) that take a DBAL connection instead of PDO.